core/src/main/java/org/jclouds/rest/internal/RestAnnotationProcessor.java

```diff
@@ -147,6 +149,7 @@ public Part apply(Entry<String, Object> from) {
    private final GetAcceptHeaders getAcceptHeaders;
    private final Invocation caller;
    private final boolean stripExpectHeader;
+   private boolean encodedParams;
```

Comment:
We should assume the annotation processor could be a singleton and avoid adding state to it. Consider removing this member variable and an alternate way to get its value in each invocation to the processor to avoid possible errors in subsequent calls due to an old state.